ksql-metastore/src/main/java/io/confluent/ksql/metastore/model/KeyField.java

```diff
@@ -79,10 +79,6 @@ private KeyField(final Optional<String> keyField, final Optional<Field> legacyKe
    * @throws IllegalArgumentException if the key is not within the supplied schema.
    */
   public KeyField validateKeyExistsIn(final Schema schema) {
-    if (keyField.isPresent() && keyField.get().equalsIgnoreCase(SchemaUtil.ROWKEY_NAME)) {
```

Comment:
i'm confused why this is being removed - does `schema` always contain ROWKEY?

Reply:
When the key is ROWKEY... yes! (Now that the implicit adding and removing has been removed from the node class)